tracker/gioui/tracker.go

```diff
@@ -165,6 +161,16 @@ func (t *Tracker) Main() {
 	t.quitWG.Done()
 }
 
+func NewWindow() *app.Window {
+	w := new(app.Window)
+	w.Option(app.Title("Sointu Tracker"))
+	w.Option(
+		app.Size(unit.Dp(800), unit.Dp(600)),
+		app.Fullscreen.Option(),
```

Comment:
Well, I don't like the fullscreen default, sorry :( Starting full screen by default is rude to me, considering the user can hit ctrl-enter to go fullscreen. No app desktop app does that, except games, and especially with the VST, there will be other windows around on the screen.

If you insist on this, please make the change of saving the window resolution and recalling that. Although tbh I don't know if there's any way to ask gio if the window is fullscreen or not; if not, then we may have to start making a user configuration file, where the user can define if fullscreen is wanted or not.